src/contacthub.js

```diff
@@ -69,17 +72,16 @@ const config = (options: ConfigOptions): void => {
 
   // set all valid option params, keeping current value (if any)
   const filteredOptions = Object.keys(options)
-    .filter(key => allowedOptions.indexOf(key) !== -1)
+    .filter(key => allowedConfigOptions.indexOf(key) !== -1)
     .reduce((obj, key) => {
       obj[key] = options[key];
       return obj;
     }, {});
   Object.assign(_ch, filteredOptions);
 
-  // default context to 'WEB', respecting cookie and options
-  if (!_ch.hasOwnProperty('context')) {
-    _ch.context = 'WEB';
-  }
+  // default values for context and contextInfo
+  _ch.context = _ch.context || 'WEB';
```

Comment:
NOTE: this is a bit different from before as with the old logic you set `context` to `'WEB'` only if the key `context` was completely missing, while now you set it to `'WEB'` if it has any falsy value (which in practice means that it's impossible for the user to force the `context` to be `undefined` or `null`)